main/production.py

```
@@ -14,9 +15,16 @@
 ]
 CORS_ALLOWED_ORIGIN_REGEXES = [
     r"^https://.*\.blob\.core\.windows\.net$",
+    r"^https://(\w+\.)*educationhub\.io$",
+    r"^http://(\w+\.)*educationhub\.io$", 
 ]
 
-
+if CUSTOM_DOMAIN:
+    ALLOWED_HOSTS.append(CUSTOM_DOMAIN)
+    CSRF_TRUSTED_ORIGINS.append('https://' + CUSTOM_DOMAIN)
+if WEBSITE_HOSTNAME:
```

Ruff (0.12.2)

`WEBSITE_HOSTNAME` may be undefined, or defined from star imports

(F405)